src/DI/AnnotationsExtension.php

```diff
@@ -54,8 +68,8 @@ public function loadConfiguration(): void
 			->setClass(Reader::class)
 			->setFactory(CachedReader::class, [
 				$this->prefix('@reflectionReader'),
-				Helpers::processCache($this, $config['cache'], 'annotations', $config['debug']),
-				$config['debug'],
+				Helpers::processCache($this, $config['cache'], 'annotations', $config['debug'] && $this->debugMode),
+				$config['debug'] && $this->debugMode,
```

Comment:
is this really how this is supposed to work in Nette 3? No way to enable debugMode for cache in production?

Reply:
You're right, stupid me - we actually do not need global debugMode at all. Changed.